diff --git a/node_modules/expo-modules-autolinking/scripts/ios/project_integrator.rb b/node_modules/expo-modules-autolinking/scripts/ios/project_integrator.rb
index 7b37f57..ba399f8 100644
--- a/node_modules/expo-modules-autolinking/scripts/ios/project_integrator.rb
+++ b/node_modules/expo-modules-autolinking/scripts/ios/project_integrator.rb
@@ -235,7 +235,9 @@ module Expo

       <<~XCODE_BUILD_SCRIPT
       # This script configures Expo modules and generates the modules provider file.
-      bash -l -c "./#{escaped_path}"
+      # Patch: use bash -c (not -l) + cd to $SRCROOT to bypass user ~/.bash_profile cd side-effects.
+      cd "$SRCROOT"
+      bash -c "./#{escaped_path}"
       XCODE_BUILD_SCRIPT
     end
